specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/apimapis.json

```diff
@@ -446,18 +446,28 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "The API was successfully deleted."
+          "202": {
+            "description": "The API was scheduled for deletion.",
+            "headers": {
+              "location": {
+                "description": "Location header",
+                "type": "string"
+              }
+            }
           },
           "204": {
             "description": "The API was successfully deleted."
           },
           "default": {
             "description": "Error response describing why the operation failed.",
             "schema": {
-              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
```

Comment:
please revert this change to avoid duplicate schema.

Reply:
Reverted.